test/test-pages/gcba/sdk-controller.ts

```diff
@@ -0,0 +1,307 @@
+/* global MediaStream */
+import GenesysCloudWebrtSdk, { ConversationUpdate, IExtendedMediaSession, ISdkConfig, ISdkConversationUpdateEvent, IStoredConversationState, SdkError, SessionTypes } from '../../../src';
+import { getInputValue, writeToLog } from './utils';
+
+let webrtcSdk: GenesysCloudWebrtSdk;
+let pendingSessions = [];
+
+let conversationUpdatesToRender: {[conversationId: string]: IExtendedMediaSession} = { };
+
+export async function initWebrtcSDK (environmentData) {
+  const jwt = getInputValue('jwt-input');
+  if (!jwt) {
+    window.alert('You need a jwt token');
+    throw new Error('Not Authenticated');
+  }
+
+  const options: ISdkConfig = {
+    jwt,
+    originAppId: 'gcba-demo',
+    originAppName: 'gcba-demo',
+    autoAcceptPendingScreenRecordingRequests: true
+  }
+
+  options.environment = environmentData.uri;
+  options.logLevel = 'info';
+  // for sumo debugging
+  // options.optOutOfTelemetry = true;
+
+  const SDK = GenesysCloudWebrtSdk;
+  (window as any).SDK = SDK;
+
+  webrtcSdk = new SDK(options);
+  (window as any).webrtcSdk = webrtcSdk;
+  (window as any).sdk = webrtcSdk;
+
+  connectEventHandlers();
+  exposeGlobalFunctions();
+
+  await webrtcSdk.initialize()
+  writeToLog(`SDK initialized with ${JSON.stringify(options, null, 2)}`);
+  renderUser(webrtcSdk._personDetails, webrtcSdk._orgDetails);
+}
+
+function connectEventHandlers () {
+  webrtcSdk.on('ready', ready);
+  webrtcSdk.on('pendingSession', pendingSession);
+  webrtcSdk.on('cancelPendingSession', cancelPendingSession);
+  webrtcSdk.on('handledPendingSession', handledPendingSession);
+  webrtcSdk.on('sessionStarted', sessionStarted);
+  webrtcSdk.on('sessionEnded', sessionEnded);
+  webrtcSdk.on('trace', trace);
+  webrtcSdk.on('sdkError', error);
+  webrtcSdk.on('disconnected', disconnected);
+  webrtcSdk.on('connected', connected);
+}
+
+function acceptPendingSession ({ conversationId }) {
+  webrtcSdk.acceptPendingSession({ conversationId });
+}
+
+function rejectPendingSession ({ conversationId }) {
+  webrtcSdk.rejectPendingSession({ conversationId });
+}
+
+function endSession ({ conversationId }) {
+  webrtcSdk.endSession({ conversationId });
+}
+
+function exposeGlobalFunctions () {
+  (window as any).acceptPendingSession = acceptPendingSession;
+  (window as any).rejectPendingSession = rejectPendingSession;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9580329)
```suggestion
  (window as unknown).rejectPendingSession = rejectPendingSession;
```